### ISSUE TITLE
[info] migration doesn't work out of the box for some

### ISSUE THREAD

**Issue by ITwrx** [kevupton/laravel-coinpayments#3]
Maybe useful info for the readme. Running the migration doesn't work out of the box with laravel 5.4+ and mariadb (as of 10.1.30-1). The laravel default DB charset is set to utf8mb4 which is too large for mariadb's varchar data type.

To fix, evidently, one could edit their AppServiceProvider.php file and inside the boot method set a default string length:

```use Illuminate\Support\Facades\Schema;

public function boot()
{
    Schema::defaultStringLength(191);
}
```

I opted to change charset and collation to 'utf8' and 'utf8_unicode_ci', respectively, in config/database.php.

thanks

**Comment by kevupton** [comment]
ahh awesome thanks, I will add a fix for this so people dont need to add this.

**Comment by kevupton** [comment]
fixed in `0.2.9`
